src/Type/Definition/InputObjectField.php

```diff
@@ -7,8 +7,10 @@
 use GraphQL\Error\Error;
 use GraphQL\Error\InvariantViolation;
 use GraphQL\Language\AST\InputValueDefinitionNode;
+use GraphQL\Type\Schema;
 use GraphQL\Utils\Utils;
 use function array_key_exists;
+use function assert;
```

Comment:
Might be unused. Wonder why it was not caught by CS check 🤔

Reply:
730e117